Jenkinsfile

```diff
@@ -52,9 +56,8 @@ pipeline {
             when { not { branch 'master' } }
             steps {
                 sh """
-                cmdstan_hash=\$(git submodule status | grep cmdstan | awk '{print \$1}')
-                bash compare-git-hashes.sh develop \$cmdstan_hash stat_comp_benchmarks
-                mv performance.xml \$cmdstan_hash.xml
+                bash compare-git-hashes.sh develop ${cmdstan_hash} stat_comp_benchmarks
```

Comment:
Need to set cmdstan_hash if it's not supplied so that the old behavior still works for branches and PRs. i.e. if cmdstan_hash == '' you need to execute `cmdstan_hash=\$(git submodule status | grep cmdstan | awk '{print \$1}') `

Reply:
got it, 1 min